docs/guide/http_cache.md

```diff
@@ -486,96 +455,281 @@ fos_http_cache:
+for cases when the limit is reached. The system will log a warning for cases where the limit is reached so you can optimize
+these cases as described above when needed.
+```yaml
+parameters:
+    # Warning, setting this means you risk losing tag information, risking stale cache. Here set below 8k:
+    ezplatform.http_cache.tags.header_max_length: 7900
+    # In order to reduce risk of stale cache issues, you should set a lower TTL here then globally (here set as 2h)
+    ezplatform.http_cache.tags.header_reduced_ttl: 7200
+```
 
-{# Or using array for several values #}
-{{ fos_httpcache_tag(['relation-33', 'relation-44']) }}
+### Response tagging done with Content View
+
+For Content views response tagging is done for you, and cache system will output headers like the following:
+```
+HTTP/1.1 200 OK
+Cache-Control: public, max-age=86400
+xkey: ez-all c1 ct1 l2 pl1 p1 p2
 ```
 
-See [Tagging from Twig Templates](http://foshttpcachebundle.readthedocs.io/en/1.3/features/tagging.html#tagging-from-twig-templates) in FOSHttpCacheBundle doc.
+If the given content have several locations you'll see several `l<location-id>` and `p<location-id>` tags in the response.
+
+
+!!! note "How response tagging for ContentView is done internally"
+
+    In `ezplatform-http-cache` there is a dedicated response listener `HttpCacheResponseSubscriber` that checks if:
+    - Response has attribute `view`
+    - View implements `eZ\Publish\Core\MVC\Symfony\View\CachableView`
+    - Cache is not disabled on the individual view
+
+    If that checks out Response will be adapted with the following:
+    - `ResponseCacheConfigurator` will apply site access settings for enabled/disabled cache and default TTL
+    - `DispatcherTagger` will dispatch the built in ResponseTaggers which will generate the tags as described above.
+
+    Further reading can be found in ezplatform-http-cache's `docs/response_taggers.md`.
+
+### Response tagging in Controllers
 
-##### In PHP
+For tagging needs in controllers, there are several options, here presented in recommended order:
 
-In PHP, FOSHttpCache exposes the `fos_http_cache.handler.tag_handler` service which enables you to add tags to a response:
+1. **Reusing DispatcherTagger to get it to pick correct tags for you**
 
+_Example for tagging everything needed for Content:_
+``` php
+/** @var \EzSystems\PlatformHttpCacheBundle\ResponseTagger\Delegator\DispatcherTagger $responseTagger */
+
+// If you have a View object you can simply call:
+$responseTagger->tag($view);
+
+// Or if you have Content / Location object only, you can instead provide content info and location:
+$responseTagger->tag($contentInfo);
+$responseTagger->tag($location);
+```
+
+2. **Use low level ContentTagInterface API for content related tags**
+
+``` php
+/** @var \EzSystems\PlatformHttpCacheBundle\Handler\ContentTagInterface $handler */
+// Specifically $handler would be instance of EzSystems\PlatformHttpCacheBundle\Handler\TagHandler
```

Comment:
I find no service defined for this.  Is it `ezplatform.http_cache.fos_tag_handler.xkey`?  Should the service to inject be mentioned?

Reply:
Well, maybe we can setup autowire alias for the interface in order to simplify this?
WDYT @alongosz ?

Reply:
> I find no service defined for this [For autowire]. Is it `ezplatform.http_cache.fos_tag_handler.xkey`? Should the service to inject be mentioned?

> True.
> 
> But maybe we can setup autowire alias for the interface in order to simplify this?

Yes. AFAICS `EzSystems\PlatformHttpCacheBundle\Handler\ContentTagInterface` can indeed safely alias `ezplatform.http_cache.fos_tag_handler.xkey` for autowiring.

Reply:
For our stuff `DispatcherTagger ` and `ContentTagInterface` => work fore me 👍 

But we have same topic here also for `3. **Manually add tags yourself using FOS TagHandler**` below, where the class has changed from FosHttpCache 1.x (`TagHandler`) to 2.x (`ResponseTagger`).

So @alongosz  fort that example maybe we should aim for mentioning service name on 2.5 doc (here), and change to autowire alias on 3.x?

Reply:
Marking this as resolved as this PR has now been updated to reflect what is _now_ proposed in PR above.